qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -486,8 +512,8 @@ def _compose_transforms(basis_transforms, source_basis, source_dag):
             source_basis but not affected by basis_transforms will be included
             as a key mapping to itself.
     """
-
-    example_gates = {(node.op.name, node.op.num_qubits): node.op for node in source_dag.op_nodes()}
+    example_gates = dict()
+    _get_example_gates(example_gates, source_dag)
```

Comment:
And the same here about constructor vs mutator.

Reply:
done